annotation/compiler/test/src/test/java/com/bumptech/glide/annotation/compiler/AppGlideModuleWithLibraryInPackageTest.java

```diff
@@ -0,0 +1,101 @@
+package com.bumptech.glide.annotation.compiler;
```

Comment:
Tests are copy-paste from `AppGlideModuleWithExcludesTest` with changes:
 * rename to classes and files to include `InPackage`
 * replace empty lib module with `LibraryModuleInPackage`
 * `:regenerateTestResources`